File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/UnknownWalletTypeException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when attempting to open a wallet with a type that has not been registered.
    /// </summary>
    public class UnknownWalletTypeException : IndyException
    {
        const string message = "The wallet type specified has not been registered.";

        /// <summary>
        /// Initializes a new UnknownWalletTypeException.
        /// </summary>
        internal UnknownWalletTypeException() : base(message, (int)ErrorCode.WalletUnknownTypeError)
        {

        }
    }

}

File: wrappers/android/src/main/java/org/hyperledger/indy/sdk/wallet/UnknownWalletTypeException.java
package org.hyperledger.indy.sdk.wallet;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Exception thrown when opening a wallet while specifying a wallet type that has not been registered.
 */
public class UnknownWalletTypeException extends IndyException
{
	private static final long serialVersionUID = -6275711661964891560L;
	private final static String message = "The wallet type specified has not been registered.";

	/**
	 * Initializes a new UnknownWalletTypeException.
	 */
	public UnknownWalletTypeException()
	{
		super(message, ErrorCode.WalletUnknownTypeError.value());
	}
}